blueprints/showcase-plugin-with-media/blueprint.json

```diff
@@ -0,0 +1,102 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Showcase plugin",
+		"description": "Offer full demo of plugin with media included",
```

Comment:
@Lovor01 what would be a better description here? You can see here how the Blueprint is shown in the gallery. 

![Screenshot 2024-12-04 at 08 27 38](https://github.com/user-attachments/assets/8b4b1333-b32c-4221-8bb7-e052a338195f)